pyanaconda/ui/gui/spokes/custom_storage.py

```diff
@@ -1857,7 +1852,6 @@ def on_add_clicked(self, button):
         # create a device of the default type, using any disks, with an
         # appropriate fstype and mountpoint
         dev_info["mountpoint"] = dialog.mountpoint
-        log.debug("requested size = %s  ; available space = %s", dialog.size, self._free_space)
```

Comment:
I guess this information is available somewhere else in the logs (or not important in this context?).

Reply:
Fixed in https://github.com/rhinstaller/anaconda/pull/2040/commits/543de98daf367130a400f2a170565a4be38e8d4d.